a3/terraform/modules/cluster/gke/kubernetes-operations/variables.tf

```diff
@@ -36,6 +36,12 @@ variable "install_nvidia_driver" {
   default     = false
 }
 
+variable "enable_ray" {
+  description = "If true will install the KubeRay operators to create a Ray cluster."
+  type        = bool
+  default     = false
+}
```

Comment:
could you make this `nullable = false` (i know we need to run through a lot of variables and add this, but it's never a bad time to start doing the right thing haha)